File: 800/2218B/main.java
import java.util.*;
public class main 
{
    public static void main(String[] args) 
    {
        Scanner sc = new Scanner(System.in);
        int t = sc.nextInt();

        while (t-- > 0) 
        {
            int sum = 0;
            int maxi = -100;

            for (int i = 0; i < 7; i++) 
            {
                int a = sc.nextInt();
                maxi = Math.max(maxi, a);
                sum -= a;
            }

            System.out.println(sum + 2 * maxi);
        }
    }
}

File: 800/2218B/main.cpp
#include<bits/stdc++.h>
using namespace std;
 
int main()
{
    int t;
    cin>>t;
    
    while(t-- > 0)
    {
        int sum = 0;
        int maxi = -100;
        
        for(int i=0;i<7;i++)
        {
            int a;
            cin>>a;
 
            maxi = max(maxi, a);
            
            sum -= a;
        }
        
        cout<<sum + 2*maxi<<endl;;
    }
    
    return 0;
}